src/test_workflow/integ_test/local_test_cluster_opensearch_dashboards.py

```diff
@@ -0,0 +1,99 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import logging
+import os
+import subprocess
+import time
+
+import psutil  # type: ignore
+import requests
+import yaml
+
+from aws.s3_bucket import S3Bucket
+from manifests.bundle_manifest import BundleManifest
+from paths.tree_walker import walk
+from test_workflow.test_cluster import ClusterCreationException, TestCluster
+from test_workflow.test_recorder.test_recorder import TestRecorder
+from test_workflow.test_recorder.test_result_data import TestResultData
+from test_workflow.integ_test.service_opensearch import ServiceOpenSearch
+from test_workflow.integ_test.service_opensearch_dashboards import ServiceOpenSearchDashboards
+
+class LocalTestClusterOpenSearchDashboards(TestCluster):
+    """
+    Represents an on-box test cluster. This class downloads a bundle (from a BundleManifest) and runs it as a background process.
+    """
+
+    def __init__(
+        self,
+        work_dir,
+        component_name,
+        additional_cluster_config,
+        bundle_manifest,
+        security_enabled,
+        component_test_config,
+        test_recorder: TestRecorder,
+        s3_bucket_name=None,
+    ):
+        self.manifest = bundle_manifest
+        self.work_dir = os.path.join(work_dir, "local-test-cluster")
+        self.component_name = component_name
+        self.security_enabled = security_enabled
+        self.component_test_config = component_test_config
+        self.bucket_name = s3_bucket_name
+        self.additional_cluster_config = additional_cluster_config
+        self.process = None
+        self.save_logs = test_recorder.local_cluster_logs
+
+        self.opensearch = ServiceOpenSearch(
+            self.work_dir,
+            self.component_name,
+            self.additional_cluster_config,
+            self.manifest,
+            self.security_enabled,
+            self.component_test_config,
+            self.save_logs,
+            self.bucket_name,
+        )
+
+        self.opensearch_dashboards = ServiceOpenSearchDashboards(
+            self.work_dir,
```

Comment:
any concern to use same work_dir for both opensearch and opensearch_dashboards

Reply:
This is the just the parent dir. Later OpenSearch related files will be put under a sub-dir with "opensearch" as prefix and OSD related files will be put under a sub-dir with "opensearch-dashboards" as prefix